### PR TITLE
Adding collation tests for MVU for 'latin1_general_ci_as' collation

### PR DESCRIPTION
**Pull request by ahmed-shameem**
The test cases contains tests cases for collation for various languages. The  tests are as follows:
1. Basic insertions in a table with English, Arabic, Mongolian, Greek, Polish, Chinese and Korean collation for varchar,
   nvarchar, char, nchar and text datatypes
2. The basic insertion is tested using ORDER BY, LIKE and WHERE clause
3. INNER JOIN using collation from same cp 
4. Test for computed column on string (SUBSTRING() only)
5. Test on PRIMARY KEY
6. For double byte characters for Chinese and Korean language, Test case for hitting truncation error is added.

Signed-off-by: Shameem Ahmed <shmeeh@amazon.com>

### Description

[Describe what this change achieves]
 
### Issues Resolved

[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/input/collation_tests-vu-prepare.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
+--varchar
+CREATE TABLE english_collation_varchar (name varchar(20) COLLATE latin1_general_ci_as);
```
Comment (Deepesh125) [inline test/JDBC/input/collation_tests-vu-prepare.sql]: Object names should be prefixed with test file name.

File: test/JDBC/input/collation_tests-vu-prepare.sql [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+GO
+
+--test for primary key
+CREATE TABLE english_collation_primary_key(name varchar(20) COLLATE latin1_general_ci_as NOT NULL PRIMARY KEY);
+GO
+
+INSERT INTO english_collation_primary_key VALUES ('new Delhi');
+GO
+INSERT INTO english_collation_primary_key VALUES ('neW DElhi');
+GO
```
Comment (Deepesh125) [inline test/JDBC/input/collation_tests-vu-prepare.sql]: Let's move this to verify script.

File: test/JDBC/input/collation_tests-vu-verify.sql [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+INNER JOIN english_collation_varchar_innerjoin ON english_collation_varchar.name = english_collation_varchar_innerjoin.name_same_cp;
+GO
+SELECT * FROM [INNER JOIN SAME CP] WHERE same_name = 'new DelHI';
+GO
+
+CREATE VIEW [INNER JOIN DIFF CP] AS
+SELECT english_collation_varchar.name AS name , english_collation_varchar_innerjoin.name_diff_cp AS diff_name
+FROM english_collation_varchar
+INNER JOIN english_collation_varchar_innerjoin ON english_collation_varchar.name = english_collation_varchar_innerjoin.name_diff_cp;
+GO
```
Comment (Deepesh125) [inline test/JDBC/input/collation_tests-vu-verify.sql]: This should go to prepare script.

File: test/JDBC/input/collation_tests-vu-verify.sql [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+SELECT substr FROM english_collation_varchar_computed_columns;
+GO
+SELECT 
+    name, 
+    collation_name 
+FROM sys.columns 
+WHERE name = N'substr';
+GO
+--SELECT cast ( case when collation_name = 'latin1_general_ci_as' then 1 else 0 end as int) as cmp from sys.all_columns where name = 'substr';
```
Comment (Deepesh125) [inline test/JDBC/input/collation_tests-vu-verify.sql]: Also, add tests to verify case [in]sensitivity of computed columns

File: test/JDBC/input/collation_tests-vu-prepare.sql (hunk 1 above)
Comment (ahmed-shameem) [reply test/JDBC/input/collation_tests-vu-prepare.sql]: Updated in the latest commit.

File: test/JDBC/input/collation_tests-vu-prepare.sql (hunk 2 above)
Comment (ahmed-shameem) [reply test/JDBC/input/collation_tests-vu-prepare.sql]: Updated in the latest commit.

File: test/JDBC/input/collation_tests-vu-verify.sql (hunk 3 above)
Comment (ahmed-shameem) [reply test/JDBC/input/collation_tests-vu-verify.sql]: Updated in the latest commit.

File: test/JDBC/input/collation_tests-vu-verify.sql (hunk 4 above)
Comment (ahmed-shameem) [reply test/JDBC/input/collation_tests-vu-verify.sql]: Updated in the latest commit.

File: test/JDBC/input/collation_tests-vu-prepare.sql [hunk 5]
```diff
@@ -0,0 +1,118 @@
+--varchar
+CREATE TABLE collation_tests_english_collation_varchar (name varchar(20) COLLATE latin1_general_ci_as);
```
Comment (surendravishnoi) [inline test/JDBC/input/collation_tests-vu-prepare.sql]: naming convention is filename_xxx, please follow for all the objects

Comment (ahmed-shameem) [reply test/JDBC/input/collation_tests-vu-prepare.sql]: Updated in the latest commit.

**Pull request closed without merging**